scipy/stats/tests/test_morestats.py

```diff
@@ -835,6 +839,56 @@ def test_1d_input(self):
         assert_raises(ValueError, stats.levene, g1, x)
 
 
+class TestFEqualityTest:
+
+    def test_data1(self):
+        args = [f1, f2]
+        T, pval = f_equality_test(*args, alternative='two-sided')
+        assert_almost_equal(T, 1.207788039, 7)
+        assert_almost_equal(pval, 0.7105517535985708, 7)
+
+    def test_data2(self):
+        args = [f1, f2]
+        T, pval = f_equality_test(*args, alternative='greater')
+        assert_almost_equal(T, 1.207788039, 7)
+        assert_almost_equal(pval, 0.6447241232007151, 7)
+
+    def test_data3(self):
+        args = [f1, f2]
+        T, pval = f_equality_test(*args, alternative='less')
+        assert_almost_equal(T, 1.207788039, 7)
+        assert_almost_equal(pval, 0.3552758767992854, 7)
+
+    def test_data4(self):
+        args = [f1, f2]
+        T, pval = f_equality_test(*args)
+        assert_almost_equal(T, 1.207788039, 7)
+        assert_almost_equal(pval, 0.7105517535985708, 7)
+
```

Comment:
```suggestion
    def test_data1(self):
        args = [f1, f2]
        T, pval = f_equality_test(*args, alternative='two-sided')
        assert_almost_equal(T, 1.207788039, 7)
        assert_almost_equal(pval, 0.7105517535985708, 7)

    def test_data2(self):
        args = [f1, f2]
        T, pval = f_equality_test(*args, alternative='greater')
        assert_almost_equal(T, 1.207788039, 7)
        assert_almost_equal(pval, 0.6447241232007151, 7)

    def test_data3(self):
        args = [f1, f2]
        T, pval = f_equality_test(*args, alternative='less')
        assert_almost_equal(T, 1.207788039, 7)
        assert_almost_equal(pval, 0.3552758767992854, 7)

    def test_data4(self):
        args = [f1, f2]
        T, pval = f_equality_test(*args)
        assert_almost_equal(T, 1.207788039, 7)
        assert_almost_equal(pval, 0.7105517535985708, 7)

```
It would be nice to `parametrize` these w.r.t. `alternative` since the only difference is the reference `p-value`. And it's even simpler, since one-sided p-values are just `p/2` or `1-p/2`, where `p` is the two-sided p-value.

See other tests that begin with `@pytest.mark.parametrize` for inspiration. Also, see `scipy/stats/tests/test_stats/__convert_pvalue_alternative`.